src/core/mount.c

```diff
@@ -1929,9 +1929,26 @@ static int mount_load_proc_self_mountinfo(Manager *m, bool set_flags) {
         return 0;
 }
 
+static int mount_enumerate_defer(sd_event_source *s, void *userdata) {
+        Manager *m = ASSERT_PTR(userdata);
+        int r;
+
+        if (!m->ready_sent)
+                return 0;
+
+        r = mount_load_proc_self_mountinfo((Manager *)userdata, false);
```

Comment:
- `userdata` -> `m`
- Please add a short comment for boolean argument: `false` -> `/* set_flags = */ false`